src/App.js

```diff
@@ -17,6 +17,7 @@ class App extends Component {
   componentDidMount(){
     fetchAllMovies()
     .then(data => this.setState({movies: data.movies}))
+    .catch((error) => this.setState({view: 'error'}))
   }
 
   switchView = (id, views) => {
```

Comment:
Oh, I love the naming for this function! Make sure the code inside only has to do with switching views. Have your App component return the main, and h1, with your switchViews called inside.